package.json

```diff
@@ -36,9 +36,12 @@
   "devDependencies": {
     "babel": "^5.2.16",
     "babel-core": "^5.2.6",
+    "babel-eslint": "^4.1.3",
     "babel-loader": "^5.0.0",
     "babel-plugin-object-assign": "^1.0.1",
     "brfs": "^1.2.0",
+    "eslint": "^1.7.3",
```

Comment:
:+1: We should definitely add an entry point for these rules within our `package.json` file; We can then extend this to use a custom Gulp task in the future if needed. Happy to merge after that change has been added.